parlai/tasks/integration_tests/agents.py

```diff
@@ -510,19 +515,14 @@ def get(self, episode_idx=0, entry_idx=0):
         return Message({'text': '1 2 3 4', field: ['1 2 3 4'], 'episode_done': True})
 
 
-class ChunkyUniqueSlowTeacher(ChunkyTeacher):
+class ChunkySlowTeacher(ChunkyTeacher):
     """
     Unique examples that load slowly.
     """
 
     def load_from_chunk(self, chunk_idx: int):
-        output = []
-        for i in range(10):
-            text = str(i + chunk_idx * 10)
-            resp = str(i + chunk_idx * 10)
-            output.append((text, resp))
         time.sleep(0.1)
```

Comment:
is adding this single time.sleep(0.1) enough?? why is it still considered a "slow" teacher? maybe make a diff teacher?

Reply:
Yes, it's enough. It's been like this for a bit.